apps/web/src/components/commands/bookmark/Config.tsx

```diff
@@ -83,6 +89,46 @@ export const BookmarkConfig = (props: CommandExtensionProps<BookmarkProps>) => {
             />
           </Popover>
         )}
+        {!isValid && (
+          <div className="flex w-full overflow-hidden rounded-md border p-4 no-underline ">
+            <Popover
+              defaultOpen={!isConfigured}
+              onOpenChange={setOpen}
+              open={isOpen}
+            >
+              <PopoverTrigger asChild>
+                <Button variant={"secondary"}>Add bookmark link</Button>
+              </PopoverTrigger>
+              <BlockConfigForm
+                align="start"
+                fields={[
+                  {
+                    name: "src",
+                    type: "text",
+                    label: "URL",
+                    defaultValue: src,
+                    placeholder: "https://",
+                  },
+                ]}
+                onSubmit={onSubmit}
+              />
+            </Popover>
+          </div>
+        )}
+        {linkPreviewQuery.isLoading && (
```

Comment:
`isIdle` also?

Reply:
It's implicitly covered by the `!isValid` check!